UnityProject/Assets/Scripts/Shuttles/ShuttleUIControl.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ShuttleUIControl : MonoBehaviour
+{
+    public MatrixMove matrixMove;
+    private float maxSpeed = 0;
+    private bool active = false;
+
+    /// <summary>
+    /// Starts or stops the shuttle.
+    /// </summary>
+    /// <param name="off">Toggle parameter</param>
+    public void TurnOnOff(bool off)
```

Comment:
Currently these only work for serverplayer, as server methods are called directly. They won't do anything on clients.
You should convert these to Commands so that they would run on server, for example:
```
[Command]
public void CmdTurnOnOff(bool off)
```
You should also add some validation to each of these (until we have our own networking layer all checks should be explicit): return if player `ps.canNotInteract() || !ps.IsInReach(position)`